app/src/main/java/com/android/shelfLife/model/recipe/ListRecipesViewModel.kt

```diff
@@ -1,158 +1,106 @@
-                  servings = 3,
-                  time = 60.minutes, // 3600 seconds = 60 minutes
-                  recipeTypes =
-                      listOf(RecipesRepository.SearchRecipeType.USE_ONLY_HOUSEHOLD_ITEMS)),
-              Recipe(
-                  name = "Tortilla de patata",
-                  instructions =
-                      listOf(
-                          instructionsTortillaDePatata), // TODO: refactor this constant into a list
-                  // of strings
-                  servings = 4,
-                  time = 90.minutes, // 5400 seconds = 90 minutes
-                  ingredients =
-                      listOf(
-                          Ingredient(
-                              FoodFacts(
-                                  name = "potato",
-                                  quantity = Quantity(amount = 4.0, unit = FoodUnit.COUNT)),
-                              true),
-                          Ingredient(
-                              FoodFacts(
-                                  name = "egg",
-                                  quantity = Quantity(amount = 8.0, unit = FoodUnit.COUNT)),
-                              true),
-                          Ingredient(
-                              FoodFacts(
-                                  name = "salt",
-                                  quantity = Quantity(amount = 15.5, unit = FoodUnit.GRAM)),
-                              true),
-                          Ingredient(
-                              FoodFacts(
-                                  name = "salt",
-                                  quantity = Quantity(amount = 20.75, unit = FoodUnit.ML)),
-                              true),
-                      ),
-                  recipeTypes = listOf(RecipesRepository.SearchRecipeType.HIGH_PROTEIN)),
-              Recipe(
-                  name = "Costillas a la brasa",
-                  instructions = listOf("cry"),
-                  servings = 3,
-                  time = 60.minutes, // 3600 seconds = 60 minutes
-                  recipeTypes =
-                      listOf(
-                          RecipesRepository.SearchRecipeType.LOW_CALORIE,
-                          RecipesRepository.SearchRecipeType.HIGH_PROTEIN)),
-              Recipe(
-                  name = "Curry rojo",
-                  instructions = listOf("cook"),
-                  servings = 4,
-                  time = 5400.seconds, // 5400 seconds = 90 minutes
-                  recipeTypes = listOf(RecipesRepository.SearchRecipeType.USE_SOON_TO_EXPIRE)),
-              Recipe(
-                  name = "Butifarra con boniato al horno",
-                  instructions = listOf("cry"),
-                  servings = 3,
-                  time = 3600.seconds, // 3600 seconds = 60 minutes
-                  // it has an extra long name to show the ... in the card
-                  recipeTypes = listOf(RecipesRepository.SearchRecipeType.LOW_CALORIE))))
+  /**
```

Comment:
Remember actually come back and do your TODOs as well as deleting certain log statements.